docs/self-managed/zeebe-deployment/operations/metrics.md

```diff
@@ -96,10 +96,12 @@ The health of partitions in a broker can be monitored by the metric `zeebe_healt
 ## Grafana
 
 Zeebe comes with a pre-built dashboard, available in the repository:
-[monitor/grafana/zeebe.json](https://github.com/camunda-cloud/zeebe/tree/develop/monitor/grafana/zeebe.json).
+[monitor/grafana/zeebe.json](https://github.com/camunda/zeebe/blob/main/monitor/grafana/zeebe.json).
 
 [Import](https://grafana.com/docs/grafana/latest/reference/export_import/#importing-a-dashboard)
 it into your Grafana instance, then select the correct Prometheus data source (important if you have more than one), and
 you should be greeted with the following dashboard:
 
-![cluster](assets/grafana-preview.png)
+![Screenshot of the grafana dashboard, showing a healthy cluster topology, general throughput metrics such handled requests and exported events per second, disk and memory usage and more](assets/grafana-preview.png)
```

Comment:
Instead of a lengthy alt text, we could always do something like the following in the text above it:

**".[Import](https://grafana.com/docs/grafana/latest/reference/export_import/#importing-a-dashboard)
it into your Grafana instance and select the correct Prometheus data source (important if you have more than one). You will then be greeted with the following dashboard, which displays a healthy cluster topology, general throughput metrics, handled requests, exported events per second, disk and memory usage, and more."**

Reply:
Makes sense :+1: I'll reduce the alt text to a minimum then, something like "Screenshot the Zeebe Grafana dashboard"

Reply:
Awesome -- adjusted this in my latest commit!

Reply:
Thanks so much, LGTM :+1: